### PR TITLE
Modify c value in PLUME schema

### PR DESCRIPTION
**Pull request by vuvoth**
# DONE 
- Modify c value in rust-k256, javascript, and rust-arkwork

### CODE REVIEW

**Comment by Divide-By-0** [comment]
The public signals that are outputted also change, did you add that?

We should really have non-zk verifiers for all of the programs as well, for testing purposes.

File: javascript/src/signals.ts [hunk 1]
```diff
@@ -77,7 +97,8 @@ export function computeS(r: Uint8Array, secretKey: Uint8Array, c: string) {
 export function computeAllInputs(
   message: string | Uint8Array,
   secretKey: string | Uint8Array,
-  r?: string | Uint8Array
+  r?: string | Uint8Array,
+  version: PlumeVersion = PlumeVersion.V1
```
Comment (rrrliu) [inline javascript/src/signals.ts:101]: confirming we want the default version to be v1?

**Review by rrrliu — COMMENTED** [review]
typescript code lgtm, with one quesiton

File: javascript/src/signals.ts (hunk 1 above)
Comment (vuvoth) [reply javascript/src/signals.ts:101]: I think this can be a latest version(V2). But I don't want to modify a old tests, so I keep it V1.

**Pull request merged**